### PR TITLE
Add more formatting

### PR DESCRIPTION
**Pull request by agude**
Change the font, text size, margins, paper size, etc.

### CODE REVIEW

**Comment by charles-uno** [comment]
Did you get a chance to look at doing a v1 build, to upload a PDF rather than a ZIP?

File: chapters/00-header.md [hunk 1]
```diff
@@ -1,5 +1,17 @@
 ---
 title: MiniFate
 author: Alexander Gude and Charles Fyfe
-classoption: twocolumn
+date: Version 0.1
```
Comment (charles-uno) [inline chapters/00-header.md:4]: 😐 

Is a version number better than the date? I think you can use `\today` to capture the build date

Comment (charles-uno) [reply chapters/00-header.md:4]: ...

**Review by charles-uno — APPROVED** [review]

File: chapters/00-header.md (hunk 1 above)
Comment (agude) [reply chapters/00-header.md:4]: I'm going to stick with version for now; dates "date" a thing. We can always switch later.

**Pull request merged**